### ISSUE TITLE
Hibernate vulnerability

### ISSUE THREAD

**Issue by blootsvoets** [OHDSI/WebAPI#1771]
Hibernate has a SQL injection vulnerability in version 5.4.21, which is resolved in version 5.4.24:
https://app.snyk.io/vuln/SNYK-JAVA-ORGHIBERNATE-1041788

**Comment by alex-odysseus** [comment]
There were a few attempts to raise the version without success:
- https://github.com/OHDSI/WebAPI/issues/1758
- https://github.com/OHDSI/WebAPI/issues/1780

We should get back to the topic when the current Spring Boot version 1.5.22.RELEASE is raised to 2.x

Until then the library spring-data-jpa-entity-graph (1.11.03) looks like a cornerstone correlating with Hibernate Core

**Comment by chrisknoll** [comment]
What is the confusion, @blootsvoets ?

**Comment by blootsvoets** [comment]
Ehm, no confusion, just sad that Spring Boot 1.5 and the latest Hibernate versions don't work together nicely.

**Comment by chrisknoll** [comment]
Agreed. For the 3.0 line we'll be bringing everything up to date: JDK 14+ etc.

**Comment by anthonysena** [comment]
Linking this to #2244
